### ISSUE TITLE
DND from mtp:// does not work (regression?)

### ISSUE THREAD

**Issue by PCMan** [lxqt/pcmanfm-qt#301]
DND from mtp:// mounted folders do not work.
DND of local files works, though.

**Comment by knokmki612** [comment]
It doesn't works for me too.

How to reproduce:
1. make mtp connection with any device
2. mount and access to device directory ![mtp device directory](https://gyazo.com/f1ee33d0e2cb2e34aed45c28a4d258c4.png)
3. drag any file and drop to desktop background from device direcory
4. It appears context menu, then click "Copy here"

What happens:
* appears copy progress dialog window, but there is no change in the progress bar.
* uri does not start from "mtp://…", just "/"
![ng](https://gyazo.com/75ce14a21d1a32f97bc2a6bb06ae5799.png)

Misc:
ctrl+c and ctrl+v from mtp device can copying successfully to local
![ok](https://gyazo.com/cda6dd55f72e5926732c3f8ec31f8815.png)

Environment:
* Gentoo Linux x86_64
* pcmanfm-qt-0.11-1

**Comment by tsujan** [comment]
I have a feeling that this can't be a problem anymore but should do tests.

**Comment by tsujan** [comment]
OK, the problem is changed:

Although this kind of DND still doesn't work, no dialog/progress is shown either -- simply nothing happens.

**Comment by tsujan** [comment]
OK, I fixed the DND with MTP by doing a part of what was done in `libfm-qt` → `utilities.cpp` for copy/paste, namely, by adding the format of "x-special/gnome-copied-files" to `FolderModel::mimeData` and retrieving it in `FolderModel::dropMimeData`.

Frankly, I still don't know why that's needed (just wild guesses) and, until I know it, making a PR won't make sense. I just saw that copy/paste worked, so I used a part of it.

The problem was that, without the above-mentioned change, file info query (`g_file_query_info`) gave the `G_IO_ERROR_NOT_MOUNTED` error (= "The specified location is not mounted") but, with it, everything went smoothly. Why? I don't know yet.
